e2e_test.go

```diff
@@ -227,3 +227,40 @@ func TestAutoIAMAuthN(t *testing.T) {
 	}
 	t.Log(tt)
 }
+
+func TestPublicIP(t *testing.T) {
```

Comment:
Shall we put region tags around this usage so we can get it into the public docs?

Reply:
Do we want to do that as a follow-up PR and arrange both Connector Public IP and Connector Auto IAM AuthN like our direct path tests?

Reply:
Because we don't currently have region tags around Connector Auto IAM AuthN...